examples/bigquery_user_info_updater_tool/cron/deployment.yaml

```diff
@@ -0,0 +1,30 @@
+# cronjob.yaml
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: bigquery-user-updater-cron
+spec:
+    schedule: "*/10 * * * *"
```

Comment:
jaketf on Aug 27, 2019 Member
Every minute seems aggressive.

 @annarudy
annarudy on Oct 25, 2019 Author Member
my bad, the 1 minute was just for testing. the actual solution this is based on was every several minutes, I want to say either 5 or 10. I will change that.